pkg/handlers/handlers.go

```diff
@@ -14,46 +17,50 @@ import (
 	"github.com/google/uuid"
 	"github.com/sirupsen/logrus"
 
-	"github.com/openshift/node-observability-agent/pkg/connectors"
 	"github.com/openshift/node-observability-agent/pkg/runs"
 	"github.com/openshift/node-observability-agent/pkg/statelocker"
 )
 
-var hlog = logrus.WithField("module", "handler")
+const (
+	ready                    = "Service is ready"
+	httpRespErrMsg           = "unable to send response"
+	timeout           int    = 35
+	logFileExt        string = "log"
+	errorFileExt      string = "err"
+	pprofFileExt      string = "pprof"
+	crioFilePrefix    string = "crio"
+	kubeletFilePrefix string = "kubelet"
+)
+
+var (
+	hlog = logrus.WithField("module", "handler")
```

Comment:
Looks like we have 2 different loggers for the operator and agent (zap vs logrus). Lets pick it separately.

Reply:
Yes, this deserves a dedicated PR.